fusor-ember-cli/app/templates/review/installation.hbs

```diff
@@ -251,6 +251,7 @@
                         value=model.cfme_install_loc }}
           {{review-link label='CFME Root password' routeName='cloudforms.cfme-configuration' isRequired=true isPassword=true value=model.cfme_root_password }}
           {{review-link label='CFME Admin password' routeName='cloudforms.cfme-configuration' isRequired=true isPassword=true value=model.cfme_admin_password }}
+          {{review-link label='CFME DB password' routeName='cloudforms.cfme-configuration' isRequired=true isPassword=true value=model.cfme_db_password }}
```

Comment:
@isratrade , thanks for catching the omission. I've added the line for the review page and tested to ensure it works as expected.